integration-cli/docker_cli_build_test.go

```diff
@@ -548,6 +548,64 @@ RUN [ $(ls -l /exists/exists_file | awk '{print $3":"$4}') = 'dockerio:dockerio'
 		build.WithFile("test_file4", "test4")))
 }
 
+func (s *DockerSuite) TestBuildCopyCacheOnFileChange(c *check.C) {
```

Comment:
All new integration tests should go into `docker_api_build...` and use the API instead of the CLI